.changeset/major-oranges-knock.md

```diff
@@ -0,0 +1,40 @@
+---
+"@neo4j/graphql": minor
+---
+
+The `where` field for nested update operations has been deprecated to be moved within the `update` input field.
+The `where` in its deprecated location is a no-op for all nested operations apart from `update`.
+
+For example, the following mutation is using the deprecated syntax:
```

Comment:
Note: This change needs to be updated in the migration guide for version 6 in the "deprecations" section